content/en/blog/2023/announcing-notation-rc3.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Notation v1.0.0-RC.3 is available!
+author:  "Notation Release Team"
+date: 2023-03-08
+draft: false
+---
+
+The Notation maintainers are pleased to announce the release of Notation v1.0.0-RC.3, including Notation CLI v1.0.0-rc.3. This blog walks you through the major updates of this release.
+
+## What's new
+
+This release introduces a breaking change to Notation:
```

Comment:
How is this a breaking change? The signatures generated from prior release will still be validated. Even signatures generated after this release can be validated by prior RC releases.

I think is a UX change, but not a breaking change.

Reply:
The main reason is the default behavior of `notation sign` command was changed. Users who execute the same command will get a different result, in this case, a different manifest stored in the registry.